src/assets/index.ts

```diff
@@ -0,0 +1,35 @@
+import type { KnownPokemonType } from "models";
+
+import bug from "./2/bug.png";
+import electric from "./2/electric.png";
+import fairy from "./2/fairy.png";
+import fighting from "./2/fighting.png";
+import fire from "./2/fire.png";
+import flying from "./2/flying.png";
+import grass from "./2/grass.png";
+import ground from "./2/ground.png";
+import ice from "./2/ice.png";
+import normal from "./2/normal.png";
+import poison from "./2/poison.png";
+import psychic from "./2/psychic.png";
+import rock from "./2/rock.png";
+import steel from "./2/steel.png";
+import water from "./2/water.png";
+
+export const typeImageMap: Record<KnownPokemonType, string> = {
```

Comment:
tip: this typing is extremely useful in Typescript: 

```ts
Record< Key, Value >
```

in this case: 

```ts
Record< ASubsetOfStringThatMeansSomethingInMyProject, string>
```

where `KnownPokemonType` is 

```ts
type KnownPokemonType =
  | "bug"
  | "electric"
  | "fairy"
  | "fighting"
  | "fire"
  | "flying"
  | "grass"
  | "ground"
  | "ice"
  | "normal"
  | "poison"
  | "psychic"
  | "rock"
  | "steel"
  | "water";
```